scripts/testnet/sovereignBridge/pyScripts/update_toml.py

```diff
@@ -68,11 +68,37 @@ def update_node_configs(config_path, esdt_prefix, sovereign_chain_address):
     update_transfer_and_execute_address(config_path + "/config.toml", sovereign_chain_address)
 
 
+def update_main_chain_elastic_url(lines, section, key, value):
+    updated_lines = []
+    section_found = False
+
+    for line in lines:
+        if line.startswith("[" + section + "]"):
+            section_found = True
+        if section_found and key in line:
+            line = re.sub(rf'({re.escape(key)}\s*=\s*)".*?"', rf'\1"{value}"', line)
+            section_found = False
```

Comment:
Why set this to False if you don't use it afterwards?

Reply:
This will skip the second `if` and will not check `key in line` every time.

Reply:
Yes, but you set it to false also in the second `if` for no reason.